### DISCUSSION TITLE
[TC] TeensCamp #2

### DISCUSSION THREAD

**Discussion by BernieHuang2008** [OpenTeens/community#1] [Ideas]
## 线上会议链接
暂无

## 基本信息
|Teens Camp|#2|
|-|-|
|类别|Advance|
|时间|3.2|
|地点|营地|
|主题|openteens论坛（前端）|

**Comment by BernieHuang2008** [comment]
[Draft]
[议程]
这玩意是格式，有关这个议程的讨论都在thread下面讨论、公布。
（回一句让我知道你们都看了？）

**Reply by BernieHuang2008** [reply]
我自己肯定看了

**Reply by Spark-Zhao666** [reply]
ok

**Comment by BernieHuang2008** [comment]
[Draft]
[议程]
可以用到MercurySQL 
~~毕竟这是我唯一一个拿的出手的东西。。。~~
客观的说，如果是用的话，还算是比较简单的，而且数据库这方面可以拓展出很多有意思的玩意

**Comment by Spark-Zhao666** [comment]
[draft]
[议程]
邀请一些鲨鱼联盟/Linkbot 的成员来讲解VEX机器人以及一些基本的控制代码和机器结构
有条件的话还可以邀请来动手实践

**Reply by BernieHuang2008** [reply]
你们用的是什么编程语言？python？

**Reply by BernieHuang2008** [reply]
thread里回复哈。。

c++对初学者可能不是那么友好叭。。？我建议初期先以Python为主

**Comment by BernieHuang2008** [comment]
[Draft]
[议程]

![IMG_20240111_214028.jpg](https://github.com/OpenTeens/community/assets/88757735/2eba7031-dd78-42e6-9602-9bae449d4313)


受一道化学题启发，我觉得可以搞一个这样的游戏。

游戏嘛，大家喜欢玩，也最有成就感。

**Comment by BernieHuang2008** [comment]
要不这样，我们做点实在的，把openteens的一些系统写写呗？
就是上次我们的白日梦，openteens账号系统、论坛、聊天系统那些

**Comment by BernieHuang2008** [comment]
Advanced 项目：openteens论坛（前端）
